syntax = "proto3";

package SSDTO;

enum Business_Type{
    LOGIN = 0;
    ENROLL = 1;
    FOWARD_MSG = 2;
    DISCONNECT = 3;
    GET_CONTACTLIST = 4;
    ADD_FRIEND = 5;
    GET_EMAILCODE = 6;
    FRIEND_SEARCH = 7;
}